packages/serde/index.ts

```diff
@@ -0,0 +1,264 @@
+// בס"ד
```

Comment:
- extract every serialize and deserialize function to it's own function.
- add a unit test for every function (Optional for this PR can be in another task)
- split every de/serialize function to it's own file by the serialization type.
- create a Factory/Builder function to get the desired Serde when used.
- add a `Readme.md` file explaining what a Serde is (GPT is great for this)